.github/workflows/pr-to-update-chart.yml

```diff
@@ -0,0 +1,65 @@
+name: PR to update Charts
+
+# This action will run after a tag starting with "v" is published
+on:
+  registry_package:
+    types: [published]
```

Comment:
I am not sure if `registry_package` published is the right trigger as this action can get triggered any time a container image is published. Like even if we push an image with `staging` tag this event will get triggered.

Reply:
So you suggest we do this with manual triggering only?